test/unit-test/cellular_pkthandler_utest.c

```diff
@@ -59,6 +60,8 @@ static uint16_t queueData = CELLULAR_PKT_STATUS_BAD_PARAM;
 static int32_t queueReturnFail = 0;
 static int32_t queueCreateFail = 0;
 static int32_t pktRespCBReturn = 0;
+static bool passCompareString = false;
```

Comment:
passCompareString should be set to 0 in the setUp() function.

Reply:
Nevermind. I see that passCompareString  is to to false in each test case when necessary.